### ISSUE TITLE
Give a Training

### ISSUE THREAD

**Issue by ethar1** [netTiers/netTiers#469]
Dear all, Looking for a professional to give a very short training course for netTiers

**Comment by jeremeguenther** [comment]
Could you give some more detail as to what you are looking for?

Are you currently using codesmith?
Are you currently using nettiers?
If so, how are you using it, if not, how do you want to use it?
Are you looking for training in how to use one of those two products, or how to design a database that works well with nettiers, or the strengths and weaknesses of nettiers?
How long are you expecting the training to be?
What is the target audience?
What is your training budget?
Have you tried contacting codesmith directly through their website to see if they offer such training?

**Comment by jeremeguenther** [comment]
Here is a link to code Smith offering just such training.
http://www.codesmithtools.com/aboutus#consulting

**Comment by niemyjski** [comment]
Just a clarification for anyone reading this. To my knowledge CodeSmith has never provided consulting or training in regards to .netTiers. We don't use .netTiers in any of our core services at CodeSmith, so we don't feel like we'd be the expertise to provide any kind of training. We've always deferred that to the netTiers project. I know in the past core contributors provided consulting at their defined rate and only if they wanted to take the work (no obligation).

We try to foster open source relationships, in the past we've hosted .netTiers websites when needed (never owned the domain name). We give out free licenses to maintainers of .netTiers and resources when we have them. For example a few years ago I added Oracle Support to the .netTiers templates.
